### ISSUE TITLE
Return the seed generated by mt.autoSeed()

### ISSUE THREAD

**Issue by laice** [ckknight/random-js#23]
Hi, is there an easy way to get the seed from the mt object after it has been set? I can't seem to find anything referencing it. Thanks for your work

**Comment by msqar** [comment]
Yea, i'm looking into that too. What I'm doing is to set the seed myself by doing a new Date().getTime() since it says it uses a date timestamp as a seed for the autoSeed(). So if I want to save the timestamp, I do it like this:

`
var nowDate = new Date().getTime();
var engine = Random.engines.mt19937().seed(nowDate);
...
`

**Comment by ckknight** [comment]
Seems like the cleanest way to handle this would be to expose the function that produces a sufficiently-entropy rich array to be able to use with the `seedWithArray` function.

**Comment by ckknight** [comment]
I am now considering this closed with 708b1ca6102fb9e80c4bead3c4516b312a4b18ed.

```
import { MersenneTwister19937, createEntropy } from 'random-js';

const seed = createEntropy();
const engine = MersenneTwister19937.seedWithArray(seed);
```

which was previously equivalent to

```
import { MersenneTwister19937 } from 'random-js';

const engine = MersenneTwister19937.autoSeed();
```

except now you can know what the initial seed was, and thus fully replay a twister using

```
const clone = MersenneTwister19937.seedWithArray(seed).discard(engine.getUseCount());
```
